app/models/device.rb

```diff
@@ -20,11 +20,11 @@ def create_dev_no
     end
 
     def add_default_image
+     device_path = default_url
```

Comment:
what is the use of this line. you can default_url, right?

Reply:
to get the path and use in to conditions to generate appropriate urls